lib/Constants.ts

```diff
@@ -450,12 +450,21 @@ export enum OAuthScopes {
 }
 
 export enum ComponentTypes {
-    ACTION_ROW  = 1,
-    BUTTON      = 2,
-    SELECT_MENU = 3,
-    TEXT_INPUT  = 4
+    ACTION_ROW         = 1,
+    BUTTON             = 2,
+    /** @deprecated Use STRING_SELECT from now on */
```

Comment:
```suggestion
    /** @deprecated Named changed. See {@link Constants~ComponentTypes.STRING_SELECT | STRING_SELECT}. This will be removed in `1.3.0`. */
```